source/_integrations/influxdb.markdown

```diff
@@ -539,6 +539,9 @@ sensor:
 sensor:
   - platform: influxdb
     api_version: 2
+    ssl: false
+    host: localhost
+    port: 9999
```

Comment:
According to our codebase, these are optional. Unless I missing a code change, if that is the case, could you point me to it?

Thanks 👍 

../Frenck

Reply:
It may be that the code defines these parameters as optional. However, without these parameters configured, the sensor definition is unable to retrieve data from the InfluxDB server.

I do not have the logs to hand - but could recreate this if needs be.

In conclusion, it sounds like a code change is required. However, the documentation change may be quicker and simpler to deploy.

Reply:
> In conclusion, it sounds like a code change is required

Sounds like a bug.

> However, the documentation change may be quicker and simpler to deploy.

That is not a solution. Please open a change to our code base or file a bug report instead.

Closing this PR in the mean time.

../Frenck